test/engine/enginerpc/enginerpc.cc

```diff
@@ -150,6 +153,182 @@ TEST_F(EngineRpc, GetVersion) {
   erpc.shutdown();
 }
 
+TEST_F(EngineRpc, GetHostsCount) {
+ std::ostringstream oss;
+  oss << "1";
```

Comment:
stringstream should be avoid when not necessary. Here, I think it is not very useful.
You can remove those two lines.